lib/datagrid/translation.rb

```diff
@@ -0,0 +1,25 @@
+require "action_view/helpers"
+
+module Datagrid
+  module Translation
+    include ActionView::Helpers::TranslationHelper
```

Comment:
I don't think we should use `TranslationHelper`. It is designed to handle view layer of rails. Using `I18n.translate` directly should give the same. Also I want to extract datagrid-core from datagrid-rails making it available for other frameworks and this part should never depend on actionview.